pkg/cfg/config.go

```diff
@@ -345,6 +351,19 @@ func (c *config) UnmarshalKey(key string, output interface{}, defaults ...Unmars
 }
 
 func (c *config) augmentString(str string) string {
+	groups := valFlagsRegexp.FindStringSubmatch(str)
+	flags := make([]string, 0)
+
+	if len(groups[2]) > 0 {
+		flags = strings.Split(groups[2], ",")
+		flags = funk.Map(flags, strings.ToLower)
+		str = groups[3]
```

Comment:
this looks unsafe, because we didn't check that the slice does have 4 elements or more in total